docs/crio.conf.5.md

```diff
@@ -1,19 +1,17 @@
-% crio.conf(5) Open Container Initiative Daemon
+% crio.conf(5) Open Container Initiative Kubernetes Container Runtime daemon
```

Comment:
Lets call this
Kubernetes Container Runtime Daemon for Open Container Initiative Containers

Reply:
Other man pages keep this short and sweet.  I'll let you all thumb wrestle over it, but I'd suggest just simply:  
 
Configuration file for CRI-O

Then go with the full name at line 6.  FWIW, in GitHub we call it "Open Container Initiative-based implementation of Kubernetes Container Runtime Interface".  I'd suggest line 6 change to:  

"configuration file for the Open Container Initiative-based implementation of Kubernetes Container Runtime Interface"

Reply:
I would rather get the Kubernetes and Container into this line so that
man -k kubernetes
man -k container

Would list cri-o.conf